content/posts/newsletter/2021-08-05.md

```diff
@@ -0,0 +1,212 @@
+
+Many thanks to the translators and their work! This is a great help to spread the news! Please join them in their work or start over with another language!
+
+## XSF Announcement
+
+Since this month a new sub-domain is available at xmpp.org: data.xmpp.org. Many thanks to MattJ! The first data project being hosted here for everyones access are the [providers lists](https://data.xmpp.net/providers/v1/providers-A.json) from the [XMPP Providers project](https://invent.kde.org/melvo/xmpp-providers). There are already first client implementation using these - please review the [criteria](https://invent.kde.org/melvo/xmpp-providers/-/blob/master/README.md) and add you service as well via the Gitlab repository! Feedback welcome!
+
+## Events
+
+[XMPP Office Hours each week](https://wiki.xmpp.org/web/XMPP_Office_Hours) - Also, checkout our new [YouTube channel](https://www.youtube.com/channel/UCf3Kq2ElJDFQhYDdjn18RuA)!
+
+[Berlin XMPP Meetup (remote)](https://mov.im/?node/pubsub.movim.eu/berlin-xmpp-meetup): Monthly Meeting of XMPP Enthusiasts in Berlin - always 2nd Wednesday of the month.
+
+## Videos
+
+XMPP Office Hours: [Building a Chat Bot on Ad Hoc Commands](https://www.youtube.com/watch?v=e1-hpsQ9OZE)
+
+## Articles
+
+The Debian XMPP Team blog [announced all the goodies](https://xmpp-team.pages.debian.net/blog/2021/07/xmpp-novelties-in-debian-11-bullseye.html) the soon to be released Debian 11 will bring. While these might not be 'new' for newsletter readers, they'll improve the experience for users of Debian Stable significantly.
+
+Seth Kenlon, from Red Hat, published two articles on the XML markup language (a quite important thing in the XMPP world ;) ) at [opensource.com](https://opensource.com). Starting with [What is XML?](https://opensource.com/article/21/7/what-xml) and following up by [Use XMLStarlet to parse XML in the Linux terminal](https://opensource.com/article/21/7/parse-xml-linux).
+
+## Software news
+
+### Clients and applications
+
+[Gajim News](https://gajim.org/post/2021-07-28-development-news-july/): Development on the new Gajim version continued in July, bringing many fixes and improvements. Also this month: WebSocket improvements and a new python-nbxmpp release.
+
+Profanity [0.11.0](https://github.com/profanity-im/profanity/releases/tag/0.11.0) is out, bringing 6 months of polishing to 0.10.0. This includes MAM support (still experimental), support for changing the password, abilities in MUCs like voice and registration, OMEMO trust mode, MUC-PMs in public channels, spam reporting, server contact discovery, and much more.
+
+[Jan-Philipp Litza](https://github.com/jplitza) and [mortzu](https://github.com/mortzu) build a XMPP feed integration for the German official warning app NINA: [Find the Github repository here](https://github.com/mortzu/nina_xmpp). One can simply [add the bot and register coordinates of interest](https://github.com/mortzu/nina_xmpp/blob/master/README.md). May it never contact you!
```

Comment:
[...] **built an** XMPP feed integration for [...]

Reply:
I always wonder: "an" would be so much nicer - but indeed, why is this allowed?